requirements.txt

```diff
@@ -1,8 +1,7 @@
 mlflow[pipelines]>=1.27.0
 pandas>=1.3.*
-pandas-profiling>=3.1.*
 scikit-learn>=1.1.*
 ipykernel>=6.12.*
 ipython>=7.32.*
 shap>=0.40.*
-jinja2==3.0.3
\ No newline at end of file
+jinja2==3.0.3
```

Comment:
These are MLP requirements included in `mlflow[pipelines]`, so I think we can remove them from the template